packages/react/__tests__/exports.ts

```diff
@@ -0,0 +1,1880 @@
+import * as exported from '..';
+
+describe('@aws-amplify/ui-react', () => {
+  describe('exports', () => {
+    it('should match snapshot', () => {
+      expect(Object.keys(exported)).toMatchInlineSnapshot(`
+Array [
+  "Alert",
+  "AmplifyChatbot",
+  "AmplifyPhotoPicker",
+  "AmplifyPicker",
+  "AmplifyProvider",
+  "AmplifyS3Album",
+  "AmplifyS3Image",
+  "AmplifyS3ImagePicker",
+  "AmplifyS3Text",
+  "AmplifyS3TextPicker",
+  "Authenticator",
+  "Badge",
+  "Box",
+  "Button",
+  "Card",
+  "Collection",
+  "CollectionTypeMap",
+  "ComponentClassNames",
+  "ComponentPropsToStylePropsMap",
+  "Divider",
+  "ErrorText",
+  "Fieldset",
+  "Flex",
+  "Footer",
+  "Form",
+  "Header",
+  "Heading",
+  "Icon",
+  "Icon10k",
```

Comment:
@dbanksdesign All of these exported icons are what I'm interested in cleaning up & instead recommending customers install their preferred icon set, but use a `className`, `data-*` attribute, or something to get the preferred style.

I'll add an engineering excellence item for it since we've had a bunch of discussion around this. I think we just need an answer for Material, heroicons, etc. & interoperability.

Reply:
Agreed, I think we need a deep dive on this. I'll add an EE item for this. Thanks for bringing this up!